speaking-portal-project/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start:dev": "nodemon",
     "start": "npm run build && node build/index.js",
-    "build": "rimraf build && tsc",
```

Comment:
Ill look further into this to see why it was causing an issue - we should remove rimraf from the dependencies if we decide its not necessary